crypto/provider_core.c

```diff
@@ -824,6 +825,42 @@ int OSSL_PROVIDER_get_conf_parameters(OSSL_PROVIDER *prov, OSSL_PARAM params[])
     return 1;
 }
 
+int OSSL_PROVIDER_conf_get_bool(const OSSL_PROVIDER *prov,
+                                const char *name, int defval)
+{
+    char *val;
```

Comment:
I would initialise this to NULL so it is always a known return - as there is currently no set-to-NULL at the start of the fetching of return pointers. It seems odd to me to pass in an address for returning data as an uninitialised value when the functions being called don't guarantee to set the value always.